IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcRelAdheresToElement/Documentation.md

```diff
@@ -0,0 +1,11 @@
+The _IfcRelAdheresToElement_ is an objectified relationship between an element and one surface feature element that that adheres to the surface on the element. The relationship is defined to be a 1 to many relationship. The _IfcRelAdheresToElement_ establishes an aggregation relationship between the main element and a sub ordinary surface feature.
```

Comment:
Suggest "The _IfcRelAdheresToElement_ is an objectified relationship between an element and **one to many surface feature elements** that **(remove one that)**"

Reply:
```suggestion
The _IfcRelAdheresToElement_ is an objectified relationship between an element and one to many surface feature elements that adhere to the surface of the element. The relationship is defined to be a 1 to many relationship. The _IfcRelAdheresToElement_ establishes an aggregation relationship between the main element and a sub ordinary surface feature.
```